databrowse/plugins/db_checklist_editor/dbs_edit_checklist.xml

```diff
@@ -145,7 +145,7 @@
                     error: saveErrorCallback,
                     processData: false,
                     success: saveSuccessCallback,
-                    type: 'POST',
+                    type: 'GET',
```

Comment:
This needs to be switched back to POST!  A GET request has limitations on parameter size, characters, and also has security implications.  

Also, you cannot upload files using GET.  This might explain why file upload doesn't work in cefdatabrowse if POST isn't currently working.  This needs to be fixed!